index.html

```diff
@@ -5061,6 +5061,143 @@ <h2 id="AudioContextInfo">
             </dl>
           </section>
         </section>
+        <section>
+          <h2 id="instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
+            The instantiation of <a>AudioWorkletNode</a> and
+            <a>AudioWorkletProcessor</a>
+          </h2>
+          <p>
+            When the constructor of <a>AudioWorkletNode</a> is invoked in the
+            main global scope, the corresponding <a>AudioWorkletProcessor</a>
+            instance is automatically created in
+            <a>AudioWorkletGlobalScope</a>. After the construction, they
+            maintain the internal reference to each other until the
+            <a>AudioWorkletNode</a> instance is destroyed.
+          </p>
+          <p>
+            When <code><a>AudioWorkletNode</a>(<em>context</em>, <em>name</em>,
+            <em>options</em>)</code> constructor is called, the user agent must
+            run the following steps in a synchronous fashion:
+          </p>
+          <ol>
+            <li>If the <code><em>context</em></code> is not an instance of <a>
+              BaseAudioContext</a>, throw a <code>TypeError</code> and abort
+              all these steps.
+            </li>
+            <li>If the <code><em>name</em></code> is not a valid
+            <code>DOMString</code> or does not exists as a key in the
```

Comment:
It's impossible for name to not be a valid DOMString; the Web IDL layer takes care of that before this algorithm is ever called.